src/common/components/share/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component, PropTypes } from 'react';
+import Clipboard from 'clipboard';
+
+import styles from './styles.css';
+
+const COPY_TO_CLIPBOARD_CLASS = styles.clipboard;
+
+export class CopyToClipboard extends Component {
+  componentDidMount() {
+    new Clipboard(`.${COPY_TO_CLIPBOARD_CLASS}`);
```

Comment:
Will it work just fine if there are two instances of that on a page?

So I suppose `new Clipboard` would be called 2 times with same class name?